From 2bae07fdbf532b75bf96b1dd144898643d57bb5f Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Wed, 12 Dec 2018 14:33:51 +0100
Subject: [PATCH 1/2] Do not record system architecture

---
 reprotest/lib/adt_testbed.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/reprotest/lib/adt_testbed.py b/reprotest/lib/adt_testbed.py
index 937b05c..3b7426f 100644
--- a/reprotest/lib/adt_testbed.py
+++ b/reprotest/lib/adt_testbed.py
@@ -61,7 +61,6 @@ class Testbed:
         self.modified = False
         self._need_reset_apt = False
         self.stop_sent = False
-        self.system_arch = None
         self.exec_cmd = None
         self.output_dir = output_dir
         self.shared_downtmp = None  # testbed's downtmp on the host, if supported
@@ -246,10 +245,6 @@ class Testbed:
 
         self.run_setup_commands()
 
-        # determine testbed architecture
-        self.system_arch = self.check_exec(self.system_interface.get_arch(), True).strip()
-        adtlog.info('testbed package architecture: ' + self.system_arch)
-
         # do we have eatmydata?
         (code, out, err) = self.execute(['which', 'eatmydata'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         if code == 0:
-- 
2.20.0.rc2

